_includes/head.html

```diff
@@ -7,15 +7,19 @@
   <link rel="stylesheet" href="{{ '/assets/css/just-the-docs-default.css' | relative_url }}">
 
   {% if site.ga_tracking != nil %}
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.ga_tracking }}"></script>
+    {% assign ga_tracking_ids = site.ga_tracking | split: "," %}
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ ga_tracking_ids.first }}"></script>
     <script>
       window.dataLayer = window.dataLayer || [];
       function gtag(){dataLayer.push(arguments);}
       gtag('js', new Date());
 
-      gtag('config', '{{ site.ga_tracking }}'{% unless site.ga_tracking_anonymize_ip == nil %}, { 'anonymize_ip': true }{% endunless %});
-    </script>
+      gtag('config', '{{ ga_tracking_ids.first }}'{% unless site.ga_tracking_anonymize_ip == nil %}, { 'anonymize_ip': true }{% endunless %});
 
+      {% for ga_property in ga_tracking_ids offset:1 %}
+        gtag('config', '{{ ga_property }}'{% unless site.ga_tracking_anonymize_ip == nil %}, { 'anonymize_ip': true }{% endunless %});
+      {% endfor %}
```

Comment:
Perhaps I'm missing something - is there a reason we couldn't instead replace this with 
```suggestion
      {% for ga_property in ga_tracking_ids %}
        gtag('config', '{{ ga_property }}'{% unless site.ga_tracking_anonymize_ip == nil %}, { 'anonymize_ip': true }{% endunless %});
      {% endfor %}
```

Reply:
Oh, now that I'm looking at the code in full context, I remember why I did it this way. Line 11 uses the first item in the array, so I got myself confused when making the code match.
https://github.com/just-the-docs/just-the-docs/blob/95e00cceb702bbd8e455f8439f0ea3c013e78aaa/_includes/head.html#L11

Still, I've now committed the simplification which still works. Thank you! 🙇